Add fonts setup as the first step in getting-started docs (#42)
warp-ds/tech-docs: docs/getting-started/index.md

This page describes how to get started building an application with Warp components.

## 1. Set up fonts

> Specify fonts with which to display text in your application. Here's an example for Finn.no:

Reviewer: I think we can add a note where to specify the fonts? I also think that we could simplify the beginning a little bit, wdyt?
```suggestion
Specify fonts to be used in your application. Here's an example for Finn.no:
```

Author: That's something I was unsure about because of some "templates" that were mentioned at our meeting. Would it be correct to say "add this to your `styles.css` file"? It should be different for each brand, so again I'm confused if that's the place.

Reviewer: Yea, I see. I would maybe prefer `global` or `base` styles. Like "add this to your base(or global) application styles".
But I think it is ok how you had it before, not mentioning the specifics. Maybe we get more info what to add in the near future
